vms/event/services.py

```diff
@@ -120,6 +120,13 @@ def get_events_by_date(start_date, end_date):
 
     return result
 
+def get_events_by_location(event_list, location):
```

Comment:
you may want to open a separate pr for this.

Reply:
I am sorry. This is a commit mistake i think. I have made a separate PR for this.

Reply:
O no problem! Just when you are submitting a change, good to club related into same pull request with separate commits, but these are completely unrelated.

Reply:
I will keep that in mind. Thank you :)